install.sh

```diff
@@ -85,55 +159,92 @@ if !([ -f /etc/ros/rosdep/sources.list.d/20-default.list ]); then
 fi
 rosdep update
 
-
-#=====================================#
-# Workspace and Sub Repository Set Up #
-#=====================================#
+instalog "Installing Sub8 ROS dependencies"
+sudo apt-get install -qq binutils-dev
+sudo apt-get install -qq ros-kinetic-ompl
+sudo apt-get install -qq ros-kinetic-spacenav-node
+
+# Get older version of Google protobuf for Gazebo
+instlog "Installing Google protobuf v2.6.1"
+wget https://github.com/google/protobuf/releases/download/v2.6.1/protobuf-2.6.1.tar.gz
+tar -xvzf protobuf-2.6.1.tar.gz
+cd protobuf-2.6.1
+./configure
+make
+make check
+sudo make install
+cd ..
+sudo rm -r protobuf-2.6.1
+sudo rm -r protobuf-2.6.1.tar.gz
 
 # Set up catkin workspace directory
 if !([ -f $CATKIN_DIR/src/CMakeLists.txt ]); then
     instlog "Generating catkin workspace at $CATKIN_DIR"
-    mkdir -p "$CATKIN_DIR/src"
-    cd "$CATKIN_DIR/src"
+    mkdir -p $CATKIN_DIR/src
+    cd $CATKIN_DIR/src
     catkin_init_workspace
-    catkin_make -C "$CATKIN_DIR"
 else
     instlog "Using existing catkin workspace at $CATKIN_DIR"
+    cd $CATKIN_DIR/src
 fi
 
-# If we're in the Semaphore-ci, we should run catkin_make in the actual build thread
-if (env | grep --quiet "SEMAPHORE=true"); then
-    mv ~/Sub8 "$CATKIN_DIR/src"
-fi
+CMAKE_PREFIX_PATH=/opt/ros/kinetic
```

Comment:
Could this potentially overwrite something important? Wouldn't it be better to just append?